mk/k3d.mk

```diff
@@ -79,8 +80,25 @@ k3d/network/create:
 		else docker network create -d=bridge $(KIND_NETWORK_OPTS) kind || true; fi && \
 		rm -f $(BUILD_DIR)/k3d_network.lock
 
+K3D_PULL_CREDENTIAL ?=
```

Comment:
Why not Always put the registry config? Just sometimes it hasn't got creds inside it...